evm_arithmetization/src/cpu/kernel/asm/beacon_roots.asm

```diff
@@ -0,0 +1,47 @@
+/// EIP-4788: Beacon block root in the EVM
+
+global set_beacon_root:
+    PUSH start_txn
+    %timestamp
+    // stack: timestamp, start_txns
+    PUSH @HISTORY_BUFFER_LENGTH
+    DUP2
+    // stack: timestamp, mod, timestamp, start_txns
```

Comment:
`mod` here is actually the HISTORY_BUFFER_LEN right? How come you prefer using `mod` in the comment and not the actual value?